app/views/catalog/_index_default.html.erb

```diff
@@ -3,18 +3,6 @@
 <% doc_presenter = index_presenter(document) %>
 <%# default partial to display solr document fields in catalog index view %>
 <% if search_results_theme == 'dc_repository_search' && home_page_theme == 'dc_repository' %>
-  <div class="document-metadata">
```

Comment:
Could you update the override comment to include why you're removing this (maybe a link to the issue?)

Reply:
Sounds good, I'll try and track this one down, this was Deon's commit so off the top of my head I'm not sure exactly what's going on yet.

Reply:
I didn't find anything that supported removing this element, I've brought it back